### PR TITLE
Fixed #248

### LINKED ISSUE #248: Alias !choice to !random

### PR DESCRIPTION
**Pull request by G-Thompson-Ops**
This contains a fix for #248 

Unsure where the readme change has come from!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,4 +31,4 @@ If you're unsure where to start, we curate a list of [good first issues](https:/
 If you have questions, you can contact us via IRC at [##zuzakistan-lab on freenode](irc://chat.freenode.net/##zuzakistan-lab)
 ([webchat](https://kiwiirc.com/nextclient/irc.freenode.net/##zuzakistan-lab)).
 
-If you encounter unacceptable behaviour in the community, please email conduct@zuzakistan.com.
+If you encounter unacceptable behaviour in the community, please email conduct@zuzakistan.com.
```
Comment (zuzak) [inline README.md]: I think this is here because you've changed whether this file ends with a newline character or not.

Comment (G-Thompson-Ops) [reply README.md]: It's the same line number though?

Comment (zuzak) [reply README.md]: image

You've removed the invisible newline character at the end. [Blame POSIX.](https://thoughtbot.com/blog/no-newline-at-end-of-file)

You can remove this file from your commit via the command line (I assume there's a similar thing in the UI):
```bash
git reset HEAD^ README.md
git commit --amend
git push origin +master
```

Comment (G-Thompson-Ops) [reply README.md]: Had to add an extra commit - let me just fix that

**Pull request merged**